app/styles/_content.scss

```diff
@@ -0,0 +1,75 @@
+.marketing{
+  height: 150px;
+}
+
+.mark-shoe{
+  top: 50%;
+  -ms-transform: translateY(-50%);
```

Comment:
Good work putting the css prefixes for cross-browser. Although I'll ask you to remove them :D. This project usas an autoprefixer task in the gulp file. What it does is to pre-process al the .css files and automatically prefix all the properties that need it.
When you're on a project that doesn't use a tool like this, you need to do it manually, but not this time.